### PR TITLE
add conditional check + lint

### PR DESCRIPTION
**Pull request by neo773**
Adds conditional check for `RECEIVER_NOT_EXPORTED`

### CODE REVIEW

File: android/src/main/java/ee/forgr/capacitor_updater/CapacitorUpdater.java [hunk 1]
```diff
@@ -257,11 +257,18 @@ private void flattenAssets(final File sourceFile, final String dest)
...
+    if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.TIRAMISU) {
+      this.activity.registerReceiver(
+          receiver,
+          new IntentFilter(DownloadService.NOTIFICATION),
+          RECEIVER_NOT_EXPORTED
+        );
+    } else {
+      this.activity.registerReceiver(
+          receiver,
+          new IntentFilter(DownloadService.NOTIFICATION)
```
Comment (riderx) [inline android/src/main/java/ee/forgr/capacitor_updater/CapacitorUpdater.java:269]: what will happen for device under android 13, the receiver is missing

Comment (riderx) [reply android/src/main/java/ee/forgr/capacitor_updater/CapacitorUpdater.java:269]: this was working before

Comment (neo773) [reply android/src/main/java/ee/forgr/capacitor_updater/CapacitorUpdater.java:269]: @riderx The receiver was added in Android 13, so don't need it for Android < 12

Comment (neo773) [reply android/src/main/java/ee/forgr/capacitor_updater/CapacitorUpdater.java:269]: @riderx 

image

Comment (riderx) [reply android/src/main/java/ee/forgr/capacitor_updater/CapacitorUpdater.java:269]: oh i didn't saw it was if else

**Pull request merged**